rfcs/0038-rosetta-construction-api.md

```diff
@@ -294,11 +320,11 @@ type payment = {
   memo: option(string),
   validUntil: option(uint32),
 };
-
-type transaction = stakeDelegation | payment
 ```
 
-Note that our client-sdk only has support for signing payments and delegations but this version of the construction API only supports those transactions types as well.
+Note that our client-sdk only has support for signing payments and delegations but this version of the construction API only supports those transactions types as well. We'll default to the client-sdk for now.
+
+Additionally, we should expose a new method in the client-sdk to feed the raw `Random_oracle.Input.t` to signer logic in addition to going through the normal flow. The Client-sdk implementation should enforce that these two implementations agree, and fail hard otherwise.
```

Comment:
For `normal flow`, you mean `signPayment` and `signStakeDelegation`?

Enforcement should be by sharing the signing code, and by CI testing.

Reply:
You're right